### PR TITLE
Testing/start cypress

### PR DESCRIPTION
**Pull request by Jayquintana**
What (if any) features are you implementing?
- create cypress files and implement end to end testing 

What (if anything) did you refactor?
- Nothing

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- keep being great

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: cypress/integration/art_view_spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+const { get } = require("http")
+
+describe('check to see if it works', () => {
+  beforeEach(() => {
+    cy.intercept('https://api.artic.edu/api/v1/artworks/search?q=painting&fields=image_id,title,date_start,date_end,artist_title,thumbnail',
```
Comment (StevieAmb) [inline cypress/integration/art_view_spec.js:5]: Awesome job with this intercept and pulling in the dummy data.

File: cypress/integration/art_view_spec.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            "height": 9067
+          },
+          "date_start": 1953,
+          "date_end": 1954,
+          "artist_title": "Mark Rothko",
+          "image_id": "df0a03a4-be1c-3f8d-73b1-5941dca57bba",
+          "title": "Untitled (Painting)"
+        }
+      ]
+    })
```
Comment (mr-ryan12) [inline cypress/integration/art_view_spec.js:23]: Thank you for figuring out how to properly stub a response! Very much appreciated!

File: cypress/integration/art_view_spec.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+  it('should display an image', () => {
+    cy.get('.featured-art').should('have.attr', 'src').should('include', 'https://www.artic.edu/iiif/2/df0a03a4-be1c-3f8d-73b1-5941dca57bba/full/843,/0/default.jpg')
+  })
+
+  it('should be able to like a piece of art', () => {
+    cy.get('.ratings-choice').first().click()
+  })
+
+  it('should display art details after click', () => {
```
Comment (StevieAmb) [inline cypress/integration/art_view_spec.js:40]: Get job with this text block and checking for everything the user should see on the page.

**Review by tonycarpenter21 — APPROVED** [review]
These tests are very thorough. Great job!

File: cypress/integration/art_view_spec.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    .get('.art-details-text').eq(0).should('have.text', 'Untitled (Painting)')
+    .get('.date-details').should('have.text', 'STARTING DATE:')
+    .get('.art-details-text').eq(1).should('have.text', '1953')
+    .get('.ending-date-details').should('have.text', 'ENDING DATE:')
+    .get('.art-details-text').eq(2).should('have.text', '1954')
+    .get('.artists-details').should('have.text', 'ARTIST:')
+    .get('.art-details-text').eq(3).should('have.text', 'Mark Rothko')
+    .get('.description-details').should('have.text', 'DESCRIPTION:')
+    .get('.art-details-text').eq(4).should('have.text', 'A work made of oil on canvas.')
+    .get('.percent-message').contains("of other people agree! Congratulations!")
```
Comment (mr-ryan12) [inline cypress/integration/art_view_spec.js:67]: Thank you Jacob! Next steps might include adding sad path testing for error handling, etc.

File: cypress/integration/main_page_spec.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+const { createYield } = require("typescript")
+
+describe('check to see if it works', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+  })
+
+  it('should confirm that true is equal to true', () => {
```
Comment (StevieAmb) [inline cypress/integration/main_page_spec.js:8]: We might want to remove the check if it's connected test from the file

File: cypress/integration/art_view_spec.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+          "image_id": "df0a03a4-be1c-3f8d-73b1-5941dca57bba",
+          "title": "Untitled (Painting)"
+        }
+      ]
+    })
+    
+    cy.visit('http://localhost:3000/painting')
+    
+  })
+  it('should confirm that true is equal to true', () => {
```
Comment (StevieAmb) [inline cypress/integration/art_view_spec.js:28]: We might want to remove the check test from the file

File: src/ArtDetails/ArtDetails.tsx [hunk 7]
```diff
@@ -28,11 +28,11 @@ const ArtDetails: React.FC<Props> = props => {
...
-      <h2>TITLE:</h2><p className="art-details-text">{props.artDetails.title}</p>
-      <h2>STARTING DATE:</h2><p className="art-details-text">{props.artDetails.date_start}</p>
-      <h2>ENDING DATE:</h2><p className="art-details-text">{props.artDetails.date_end}</p>
-      <h2>ARTIST:</h2><p className="art-details-text">{props.artDetails.artist_title}</p>
-      <h2>DESCRIPTION:</h2><p className="art-details-text">{props.artDetails.thumbnail.alt_text}</p>
+      <h2 className='title-details'>TITLE:</h2><p className="art-details-text">{props.artDetails.title}</p>
+      <h2 className='date-details'>STARTING DATE:</h2><p className="art-details-text">{props.artDetails.date_start}</p>
+      <h2 className='ending-date-details'>ENDING DATE:</h2><p className="art-details-text">{props.artDetails.date_end}</p>
+      <h2 className='artists-details'>ARTIST:</h2><p className="art-details-text">{props.artDetails.artist_title}</p>
+      <h2 className='description-details'>DESCRIPTION:</h2><p className="art-details-text">{props.artDetails.thumbnail.alt_text}</p>
```
Comment (mr-ryan12) [inline src/ArtDetails/ArtDetails.tsx:35]: Excellent way to test for elements by targeting them by className!

File: cypress/integration/main_page_spec.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+  it('should display a page title', () => {
+    cy.get('.logo').should('have.text', 'Art From TheHeart')
+  })
+
+  it('should display a logo', () => {
+    cy.get('.logo-icon').should('exist')
+  })
+
+  it('should display a page welcome message', () => {
```
Comment (StevieAmb) [inline cypress/integration/main_page_spec.js:20]: Great job!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**